### ISSUE TITLE
Bug rotate log on startup

### ISSUE THREAD

**Issue by textanalyticsman** [oracle/weblogic-deploy-tooling#938]
We are using below properties

```
            WebServer:
                WebServerLog:
                    FileCount: 25
                    # TODO: Below line is not applied, we should check why
                    RotateLogOnStartup: true
                    LogFileFormat: extended
```
However, we get this

![image](https://user-images.githubusercontent.com/25264959/124781912-164f5480-df44-11eb-99a0-dd64f4cc23e6.png)

Kindly help us to fix it.

Best regards,
TAM

**Comment by CarolynRountree** [comment]
can you verify which version of WDT you are using please?

**Comment by CarolynRountree** [comment]
I am unable to recreate this using WLS 12.2.1.3

**Comment by ddsharpe** [comment]
@textanalyticsman I cannot find anything that would cause the behavior that you describe.
If I run 
``` bash
./createDomain.sh -domain_parent . -model_file ./rotateLog.yaml
```
using the model
``` yaml
domainInfo:
  AdminPassword: welcome1
topology:
  Name: rotatelog
  Server:
    AdminServer:
      WebServer:
        WebServerLog:
          RotateLogOnStartup: true
```
I get config.xml like:
``` xml
  <server>
    <name>AdminServer</name>
    <web-server>
      <name>AdminServer</name>
      <web-server-log>
        <name>AdminServer</name>
      </web-server-log>
    </web-server>
```
The `Rotate log file on startup` checkbox on the WebLogic Console shows as checked (on/true) for AdminServer -> Logging -> HTTP.

Flipping the value to false in the YAML and repeating the same process:
``` yaml
          RotateLogOnStartup: false
```
I see the the config.xml looks correct, as well does the console page with the checkbox unchecked (off/false).
``` xml
  <server>
    <name>AdminServer</name>
    <web-server>
      <name>AdminServer</name>
      <web-server-log>
        <name>AdminServer</name>
        <rotate-log-on-startup>false</rotate-log-on-startup>
      </web-server-log>
    </web-server>
```

Please reopen this bug if you believe there is still an issue here.  For now, I am closing this issue as not reproducible.

**Comment by ddsharpe** [comment]
@textanalyticsman This is a WLST offline bug in WebLogic.  The bug is fixed in the next production release of WebLogic Server.  To get this fix for the release of WLS that you are using, you will need to file a support case with Oracle technical support and ask that the fix be ported to your specific release.  Be sure to mention, in your support case, all one-off patches and PSUs that you are using. The bug number of the original fix is 29547985.
